packages/zoe/test/unitTests/contracts/newSwap/test-newSwap-swap.js

```diff
@@ -907,34 +907,3 @@ test('newSwap jig - swapOut uneven', async t => {
 // This demonstrates that we've worked around
 // https://github.com/Agoric/agoric-sdk/issues/3033.  When that is fixed, the
 // work-around in collectFees should be cleaned up.
-test('newSwap workaround zoe Bug', async t => {
```

Comment:
Hmm, not thinking about any XS concerns, shouldn't we keep this test around? I think it's good to know that Zoe handles this correctly, and ensure it will still handle it correctly even if we are refactoring Zoe code.

Reply:
Reinstated and updated the test and updated the comment.